pynopegl-utils/pynopegl_utils/viewer/__init__.py

```diff
@@ -291,22 +291,35 @@ def _export_video(self, filename: str, res_index: int, profile_index: int, sampl
 
             fd_r, fd_w = os.pipe()
 
+            ffmpeg = ["ffmpeg"]
+            input = f"pipe:{fd_r}"
+
+            if platform.system() == "Windows":
+                import msvcrt
+
+                handle = msvcrt.get_osfhandle(fd_r)
+                os.set_handle_inheritable(handle, True)
+                input = f"handle:{handle}"
+                ffmpeg = [sys.executable, "-m", "pynopegl_utils.viewer.ffmpeg_win32"]
+
             # fmt: off
-            cmd = [
-                "ffmpeg", "-r", "%d/%d" % fps,
+            cmd = ffmpeg + [
+                "-r", "%d/%d" % fps,
                 "-v", "warning",
                 "-nostats", "-nostdin",
                 "-f", "rawvideo",
                 "-video_size", "%dx%d" % (width, height),
                 "-pixel_format", "rgba",
-                "-i", "pipe:%d" % fd_r,
-            ] + profile.args + [
+                "-i", input] + profile.args + [
```

Comment:
weird indent: can you keep it the way it was before? The `] + profile.args + [` should remain unchanged afaict

Reply:
Done.